modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/export/api/AbstractExporter.java

```diff
@@ -67,4 +69,9 @@ public boolean displayOn(String page) {
         return descriptor.displayOn(page);
     }
 
+    protected boolean isPrettyPrint(Map<String, String> properties, String defaultValue) {
+        return Boolean.valueOf(getProperties().getOrDefault(PRETTY_PROPERTY, defaultValue))
+                || (properties != null && Boolean.valueOf(properties.getOrDefault(PRETTY_PROPERTY, defaultValue)));
```

Comment:
It seems this code was just moved but you could cleanup `valueOf` -> `parseBoolean`